client/blocks/feedback/edit.scss

```diff
@@ -4,6 +4,22 @@
 
 	.wp-block[data-type="crowdsignal-forms/feedback"] {
 		width: auto;
+		// This number is was carefully chosen so the block and its overlay
```

Comment:
Present past contiguous tense is not allowed!

Reply:
Don't give them any more ideas :joy: